docs/source/faq.rst

```diff
@@ -494,3 +494,33 @@ There are two kinds of constrained optimizations, one with soft constraints and
 Soft constraints do not have to be satisfied, but an objective function is penalized if they are unsatisfied. On the other hand, hard constraints must be satisfied.
 
 Optuna is adopting the soft one and **DOES NOT** support the hard one. In other words, Optuna **DOES NOT** have built-in samplers for the hard constraints.
+
+How can I parallelize optimization
+-------------------------------------------------
+
+The variations that can be expected to be parallelized are in the following three cases.
+
+1. one-node, multi-threading parallelization
+2. one-node, multi-processing parallelization
+3. multi-node, multi-processing parallelization
+
+## For the case of 1
+Parallelization can be achieved by setting the argument n_jobs in the optimize method of the Study class.
+However, since it is multi-threading, the python code will not be faster due to GIL. 
+
+For more information about 1., see APIReference.
+.. APIReference: https://optuna.readthedocs.io/en/stable/reference/index.html
```

Comment:
You can check the link is generated by checking the ci of `ci/circleci: document artifact` e.g. 

https://68543-122299416-gh.circle-artifacts.com/0/docs/build/html/index.html

Reply:
To add links, you need to write those things: https://planset-study-sphinx.readthedocs.io/ja/latest/04.html#id14

```
* Sphinx-users.jp_

.. _Sphinx-users.jp: http://sphinx-users.jp/
```

Reply:
Thank you for pointing this out. The correction has been addressed.